test/test_amity.py

```diff
@@ -0,0 +1,415 @@
+import unittest
+from office_space_allocation import amity, person, fellow, staff, room, office, livingroom
+from office_space_allocation.utilities import InvalidRoomOccupantError, RoomFullError
+from office_space_allocation import db
```

Comment:
Arrange your imports in alphabetical order i.e python libraries should start in their own alphabetical order then have an empty line and add those related to office allocation afterwards in their alphabetical order.